src/v/pandaproxy/schema_registry/sharded_store.cc

```diff
@@ -198,6 +198,17 @@ sharded_store::project_ids(subject_schema schema) {
       v_id.value_or(invalid_schema_version), s_id.value(), is_new};
 }
 
+ss::future<bool> sharded_store::upsert(
+  seq_marker marker,
+  unparsed_schema schema,
+  schema_id id,
+  schema_version version,
+  is_deleted deleted) {
+    // NOLINTNEXTLINE(bugprone-use-after-move)
+    auto canonical = co_await make_canonical_schema(schema);
+    co_return co_await upsert(marker, canonical, id, version, deleted);
```

Comment:
What was throwing the use-after-move error?

Reply:
Maybe it was cooy-pasta, removed.